Build.ps1

```diff
@@ -0,0 +1,100 @@
+Param(
+    # Archive containing the Ceph Windows binaries, will be fetched using scp.
+    # Can be a local path, a UNC path or a remote scp path.
+    [Parameter(Mandatory=$true)]
+    [string]$CephZipPath,
+
+    # The thumbprint of the X509 certificate used for signing the WNBD driver
+    # and the MSI installer
+    [string]$SignX509Thumbprint = $null
+)
+
+$ErrorActionPreference = "Stop"
+
+function SetVCVars($version="2019", $platform="x86_amd64") {
+    pushd "$ENV:ProgramFiles (x86)\Microsoft Visual Studio\$version\Community\VC\Auxiliary\Build"
+    try {
+        cmd /c "vcvarsall.bat $platform & set" |
+        foreach {
+          if ($_ -match "=") {
+            $v = $_.split("="); set-item -force -path "ENV:\$($v[0])"  -value "$($v[1])"
+          }
+        }
+    }
+    finally {
+        popd
+    }
+}
+
+function Sign($x509thumbprint, $path) {
+    & signtool.exe sign /sha1 $x509thumbprint /td SHA256 /v $path
+    if($LASTEXITCODE) { throw "signtool failed" }
+}
+
+function BuildWnbd() {
+
+    pushd $depsBuildDir
+    & git.exe clone https://github.com/cloudbase/wnbd
+    if($LASTEXITCODE) {
+        throw "git failed"
+    }
+    cd wnbd
+
+    & msbuild.exe vstudio\wnbd.sln /p:Configuration=Release
+    if($LASTEXITCODE) {
+        throw "msbuild failed"
+    }
+
+    if($SignX509Thumbprint) {
+        Sign $SignX509Thumbprint .\vstudio\x64\Release\driver\wnbd.sys
+        Sign $SignX509Thumbprint .\vstudio\x64\Release\driver\wnbd.cat
+    }
+
+    copy vstudio\x64\Release\driver\* ..\..\Driver\
+    copy vstudio\x64\Release\libwnbd.dll ..\..\Binaries\
+
+    popd
+}
+
+function GetCephBinaries() {
+    pushd $depsBuildDir
+
+    & scp.exe $CephZipPath ceph.zip
+    if($LASTEXITCODE) {
+        throw "scp failed"
+    }
+
+    if (Test-Path ceph) {
+        rm -Recurse -Force ceph\
+    }
+
+    Expand-Archive ceph.zip -DestinationPath .
+    rm ceph.zip
+    copy -Path ceph\* -Destination ..\Binaries\ -Exclude README.windows.rst
+

```

Comment:
Currently, ceph.zip contains everything built during the ceph build process. We should either leave out the unnecessary bits (such as ceph_test*, unittest_*, etc.) in the original ceph.zip creation, or limit the files we add to the installer build here.

Reply:
@ml8mr this was exactly what sparkled the whole conversation on this topic as my ceph.zip was full of those *test*.exe even if I was building with SKIP_TESTS=1, then I found out that the issue was here: petrutlucian94/ceph#8.
Once I found that out, I removed the filter here, but we could readd it anytime

Reply:
Hmm, the readme was meant to be included, might be a good idea to keep it: https://github.com/petrutlucian94/ceph/commit/dde57f7499a487092edc70abe824c6391654f977#diff-d6e26fbb9d95cc194c0098cb59dcd228R203

We can use ``-Exclude "*test*"`` instead.

Reply:
Limiting list of exes as per last conversation